EnglishTranslation-master/QUEST.tsv

```diff
@@ -1023,13 +1023,13 @@ QUEST_20150317_001022	I guess the real revelation is in the chapel? {nl}That Lai
 QUEST_20150317_001023	This is the Seal of Space...{nl}
 QUEST_20150317_001024	We will need the Revelator.
 QUEST_20150317_001025	The Revelator has came here alone. {nl}Exactly what I wanted.
-QUEST_20150317_001026	For what purpose did you come to this cursed city? {nl}Please go back if you don't have a particular purpose.
+QUEST_20150317_001026	For what purpose did you come to this cursed city? {nl}Please go back if you don't have a particular reason.
 QUEST_20150317_001027	So you are the famous Revelator. The Knights of Kaliss are looking for competent people like you. {nl}If you're here to help them, I'll temporarily permit your entrance.
 QUEST_20150317_001028	But please don't forget. The right to enter here belongs to our soldiers. {nl}We hope you don't create unnecessary trouble.
 QUEST_20150317_001029	Salister... What are you even talking about? {nl}That's absurd...
 QUEST_20150317_001030	The Symbol of Silt is on my hands thanks to you. {n}If I pass this symbol to the dumb Royal Army or the Knights of Kaliss, they're going to give me more shifts on watch.
 QUEST_20150317_001031	Only an ambitious and clever man such as I could unleash this object's true potential.
-QUEST_20150317_001032	I will use the power of Petrification and Ice to awaken the ancient power that existed hundreds years ago. {nl}You just stand there and watch.
+QUEST_20150317_001032	I will use the power of Petrification and Ice to awaken the ancient power, power that existed hundreds years ago. {nl}You just stand there and watch.
```

Comment:
I prefer the older version as power is used 3 times in your version.

We can shorten it to:
I will now use the power of Petrification and Ice to awaken the ancient power from centuries ago. {nl}Please stay there and observe.